packages/integration-sdk-core/src/types/logger.ts

```diff
@@ -10,10 +10,15 @@ interface ChildLogFunction {
   (options: object): IntegrationLogger;
 }
 
+export interface StepLogAdditionalContext {
+  parentStep?: StepMetadata;
+}
+
 type StepLogFunction = (step: StepMetadata) => void;
 type StepLogFunctionWithReason = (
   step: StepMetadata,
   reason: DisabledStepReason,
+  additionalContext?: StepLogAdditionalContext,
```

Comment:
Instead of adding additionalContext as a new param, could we add an addition type to reason?

DisabledStepReasonWithContext

And that way PARENT_DISABLED could require the additional context.

This would prevent a scenario where the string built does not have the parent's name.

Not hard set on this, thoughts from you?

Reply:
```
type DisabledParentStepReason {
 reason: DisabledStepReason.DISABLED_PARENT
 parentStepName: string
}

```

Reply:
Since this is an exposed function and other repos are using it, I wanted to implement something to have the less possible impact and I think adding an additional optional param would not affect the existing repos using this function instead of modifying one of the existing params (it could potentially have any impact in the clients). What do you think? @VDubber

Reply:
I see your point. Maybe we should just validate the parent step name and throw if it isn't present.